docs-source/spring/content/platform/vault/_index.md

```diff
@@ -234,4 +234,22 @@ For more information about the Key/Value secrets engine [Key/value Documentation
 
     <!-- spellchecker-disable -->
     {{< img name="vault-show-secret" size="medium" lazy=false >}}
-    <!-- spellchecker-enable -->
\ No newline at end of file
+    <!-- spellchecker-enable -->
+
+## Accessing Vault Recovery Keys
+
+Vault is configured to Auto Unseal using OCI Vault. Initializing with Auto Unseal creates 5 recovery keys and they are stored in k8s secrets. They **MUST** be retrieved and stored in a 2nd location.
+
+To extract the 5 recovery keys use the following commands:
```

Comment:
In formal English, numbers under 20 are normally spelled using letters, not numerals - so "second" and "five"